File: app/src/main/java/org/qiah/balabala/myApplication.java
package org.qiah.balabala;

import androidx.annotation.NonNull;

public class myApplication extends BaseApplication {
    @NonNull
    @Override
    public String iconFontPath() {
        return "iconfont.ttf";
    }

    @Override
    protected String normalPath()  {
        return "sourcehansans.ttf";
    }
}

File: app/src/main/java/org/qiah/balabala/myApplication.kt
package org.qiah.balabala

import org.qiah.balabala.SQLite.MyDataBaseHelper

class myApplication : BaseApplication() {
    override fun onCreate() {
        super.onCreate()
        MyDataBaseHelper(context(), "Nikke_balabala", 1)
    }

    public override fun iconFontPath(): String {
        return "iconfont.ttf"
    }

    override fun normalPath(): String {
        return "SourceHanSansCN-Normal#1.otf"
    }
    override fun blodPath(): String {
        return "SourceHanSansCN-Heavy#1.otf"
    }
}
